Source/missiles.h

```diff
@@ -98,6 +98,7 @@ void AddDiabApoca(int mi, int sx, int sy, int dx, int dy, int midir, char mienem
 int AddMissile(int sx, int sy, int dx, int dy, int midir, int mitype, char micaster, int id, int midam, int spllvl);
 int Sentfire(int i, int sx, int sy);
 void MI_Dummy(int i);
+void MI_Dummy2();

```

Comment:
```suggestion
void dummy_nop_used_in_dx_init();
```

Also, move this to `dx.h` for now, to follow the convention used in `multi.h`.